how-to/customize-workspace/scripts/pack.json

```diff
@@ -2,6 +2,9 @@
 	"description": "This is a cut down list of the dependencies that we want to package. This is used by the package-content script which looks at paths specified in package and tries to resolve them (in the same way it reads a manifest). The /package entry is a virtual url used by package-config to fetch specific assets.",
 	"package": [
 		"http://localhost:8080/platform/provider.html",
+		"http://localhost:8080/shell/shell.html",
```

Comment:
Where did the pack.json com from ? Can it be renamed, I was very confused as to what it was doing